npm/grep/src/support.js

```diff
@@ -228,7 +228,7 @@ if (!Cypress.grep) {
    *  // remove all current grep settings
    *  // and run all tests
    *  Cypress.grep()
-   * @see "Grep from DevTools console" https://github.com/cypress-io/cypress-grep#devtools-console
```

Comment:
[system-tests/projects/nx-monorepo](https://github.com/cypress-io/cypress/pull/26108/files#diff-b09164f9746e441615bb535496c1fdcbb9e4a7621866049d8a4ec3fa863f6a6e) was this file added on accident?

Reply:
Fixed